### PR TITLE
[navbar + Hero ] - Implemented `navbar` and `hero` sections

### PR DESCRIPTION
**Pull request by asadfma**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Landify Project</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
```
Comment (abanseka) [inline index.html]: - Add the font used here (inter) from google fonts

File: index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+         <a href="#"><img src="./images/Google Play Badge.png" alt="Google Badge">
+         </a>
+       </li>
+       <li>
+         <a href="#">
+            <img src="./images/App Store Badge.png" alt="Apple store Badge">
+         </a>
+        </li>
+     </ul>
+    </nav>
```
Comment (abanseka) [inline index.html]: Properly structure this
```
<nav>...</nav>
<section class="hero-section">...</section>
```

- Give meaningful class names to your elements. the names should kinda describe the elements

File: index.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+         Organise projects. <br>
+         Get more done.
+       </h1>
+       <button>
+         Get Started
+       </button>
+     </span>
+     <img src="./images/Phone Mockup.png" alt="First photo Mockup">
+    </div>
+ </header>
```
Comment (abanseka) [inline index.html]: same here
- Give class names with meaning
- From what we discussed in the meeting. think about structure first and use CSS to enhance

File: style.css [hunk 4]
```diff
@@ -0,0 +1,95 @@
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body{
+  margin: 0 auto;
+}
```
Comment (abanseka) [inline style.css]: - Move ` *{ ... } ` to a separate `reset.css` file
- the reset already has `margin:0`. it's not necessary within  `body`

File: style.css [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  border-radius: 10px 200px;
+  color: #ffffff;
+  background-color: rgb(8, 115, 207);
+  cursor: pointer;
+}
+.down img{
+filter: drop-shadow(10px 15px 30px rgba(41, 41, 42, 0.5));
+margin-top: 100px;
+margin-right: -20px;
+}
```
Comment (abanseka) [inline style.css]: - Use the meaningful class names that you will give in the HTML
- Don't apply styles directly to any tag 
> as discussed do research on the BEM CSS convention
- avoid using absolute units on macro-level elements (it's like hard coding them and they won't change regardless of the screen size) we discussed it.

- Use the font defined in the design
- Ensure that you are correctly following the design specification

**Review by abanseka — CHANGES_REQUESTED** [review]

**Comment by abanseka** [comment]
- Pls don't add spaces in the names of your images. separate them with an _ or - 
_Compliant_: :heavy_check_mark: 
- some-image-name.jpg
- some_image_name.jpg

_Non-Compliant_: :no_entry: 
-some image name.jpg
-some-image_name.jpg

File: index.html (hunk 1 above)
Comment (asadfma) [reply index.html]: okay , you can check, i have added only SELECT EXTRALIGHT 200 and Regular 400 https://prnt.sc/uUL38cb84Xg-

File: index.html (hunk 2 above)
Comment (asadfma) [reply index.html]: yooo, bro you know how challenging it is. especially if you are starter.

File: index.html (hunk 3 above)
Comment (asadfma) [reply index.html]: i did, please can you check them one by one again if its considerable

File: style.css (hunk 5 above)
Comment (asadfma) [reply style.css]: 1. when it comes to the, bem css convention after setting up in my vs code yesterday i couldnt figure out using it properly and did nt work
2. so, everything shown in the figma i shall follow it .
3. i have changed the font-sizes here in header section

**Comment by asadfma** [comment]
Pls don't add spaces in the names of your images. separate them with an _ or -
Compliant: ✔️
some-image-name.jpg
some_image_name.jpg

 understood brother,

**Pull request merged**